src/lv_widgets/lv_keyboard.c

```diff
@@ -354,6 +400,14 @@ void lv_keyboard_def_event_cb(lv_obj_t * kb, lv_event_t event)
         lv_btnmatrix_set_ctrl_map(kb, kb_ctrl[LV_KEYBOARD_MODE_TEXT_LOWER]);
         return;
     }
+    #if LV_USE_ARABIC_PERSIAN_CHARS == 1
+    else if(strcmp(txt, "أب") == 0) {
+        ext->mode = LV_KEYBOARD_MODE_TEXT_UPPER;
```

Comment:
Why set it to `UPPER` when `LV_KEYBOARD_MODE_TEXT_ARABIC` exists?

Reply:
sorry that is a mistake
I don't  know how I didn't notice that